translate/cloud-client/beta_snippets_test.py

```diff
@@ -89,10 +89,10 @@ def test_list_languages(capsys):
 
 
 def test_list_languages_with_target(capsys):
-    beta_snippets.list_languages_with_target(PROJECT_ID, 'is')
+    beta_snippets.list_languages_with_target(PROJECT_ID, 'es')
     out, _ = capsys.readouterr()
-    assert u'Language Code: sq' in out
-    assert u'Display Name: albanska' in out
+    assert u'Language Code: en' in out
+    # assert u'Display Name: anglais' in out
```

Comment:
Same for this one.

Reply:
Also do all these languages and results have to be changed or just the ones that are actually failing?

Reply:
See above.

Reply:
Can you clarify what you mean about "all these languages"?

Reply:
Most of these tests are passing fine as is without any changes. Only modify failing tests such as:
`test_translate_text` and `test_translate_utf8`

Where mostly just the results have been modified from likely endpoint changes.

Reply:
For me, test_detect_language, test_list_languages_with_target, and test_translate_text were failing. I've changed test_translate_with_glossaries to better test a fix I made to the src.

Reply:
Do you have the latest source on the master branch? I'm checking against that. 

Might be a python2 vs. python3 thing.

Reply:
Alright, looks like python2 doesn't support all these. 
Should only need to change the one in `test_detect_language`.

Reply:
[resolving with most recent commit]